connector/checkpoint/checkpoint_view.xml

```diff
@@ -48,6 +48,7 @@
                         delete="false" edit="false" version="7.0">
                     <field name="model_id"/>
                     <field name="record"/>
+                    <field name="message"/>
```

Comment:
well, it depends on what you are tracking. For instance, in the PS connector you can track webservice errors and that could be a long string, but in this case you don't have record and model.
You know what? I don't think it's a real issue with modern screen sizes, so maybe we can just try this way and if in "real life" we recognize is not good we can always find a better solution. Make sense?